doc/user/overview/configuration.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Configuring Materialize"
+description: "Understand the command line arguments that control Materialize's behavior"
+menu:
+  main:
+    parent: 'overview'
+    weight: 2
+---
+
+All of `materialized`'s command line options can be viewed by running `materialized --help`. This page lists additional considerations and best practices when running Materialize.
+
+## Command line flags
+
+### Data directory
+
+Upon startup `materialized` creates a directory where it persists metadata. By default, this directory is called `mzdata` and is situated in the path where you ran `materialized`. Currently, only metadata is persisted in `mzdata`. You can specify a different directory using the `--data-directory` flag. Upon start, `materialized` checks for an existing data directory, and will reinstall source and view definitions fro it if one is found.
```

Comment:
and is situated in the current working directory of the materialized process.

Reply:
Done.